src/components/LoginComponent.js

```diff
@@ -0,0 +1,66 @@
+
+import React, { Component } from 'react';
+import '../Css/style.sass';
+import '../Css/App.css';
+import '../Css/style.css';
+//import loginJS from './login';
+//import $ from 'jquery';
+
+class LoginComponent extends Component {
+
+   constructor(props) {
+       super(props);
+
+       this.state = {
+           value: '',
+           submittedValue: ''
+       }
+
+       this.onChangeInput = this.onChangeInput.bind(this);
+       this.onSubmitForm = this.onSubmitForm.bind(this);
+   }
+
+   onChangeInput(event) {
```

Comment:
where is this used

Reply:
Not 100% Sure 
our team mate isn't here so we just added his work to our project and we will probably strip that out.